templates/influxdb.conf.erb

```diff
@@ -23,7 +24,7 @@ reporting-disabled = <%= @reporting_disabled %>
   wal-partition-flush-delay = "<%= @wal_partition_flush_delay %>"
   wal-dir = "<%= @wal_dir %>"
   wal-logging-enabled = <%= @wal_logging_enabled %>
-  data-logging-enabled = <%= @data_logging_enabled %>
+  trace-logging-enabled = <%= @data_logging_enabled %>
```

Comment:
I think it would be confusing/wrong/bad to have a @data_logging_enabled control a trace-logging-enabled parameter ( emphasising the _data_ prefix )